AutoDuty/Managers/ActionsManager.cs

```diff
@@ -207,7 +209,7 @@ private bool InteractableCheck(IGameObject? gameObject)
             if (EzThrottler.Throttle("Interactable", 250))
             {
                 if (ObjectHelper.GetBattleDistanceToPlayer(gameObject) > 2f)
-                    MovementHelper.Move(gameObject, 0.25f, 2f, false);
+                    MovementHelper.Move(gameObject, 1.5f, 2f, false);
```

Comment:
I get what the rest is doing by why are we changing the Tollerance of the Movement here?

Reply:
because most interaction spots actually have some form of collision, and Interact failed a lot because it tried to path inside these objects and never managed to reach it.

Reply:
That definitely should not be happening, the code is actually supposed to just be moving into interact range, which is within 2, including the player and the objects hitboxes, which in most cases is actually around 3, if it is trying to get closer in any situations there is a bug, is there a specific interactable that this happens on I will investigate further!

Reply:
easiest test would be Praetorium, the very first interaction console.
The current path in the repo circumvents it by pathing to the console in a separate step, so the line of code I changed is never run.

Reply:
I do see what it's doing and i see the bug, I fixed it. I will merge yours first then put my fix on top